schemas/test/en/test_placeholder_first_non_empty_item_repeating_sections.json

```diff
@@ -0,0 +1,398 @@
+{
```

Comment:
This schema doesn't test repeating sections. It only adds items. The list collector page isn't a repeating section. To test repeating section, you need to add a section that repeats over the list of items.